book/src/config/partial.md

```diff
@@ -47,3 +47,26 @@ As stated above, partials also handle the following:
 - Merging partials with [strategy functions](./struct/merge.md).
 - Validating current values with [validate functions](./struct/validate.md).
 - Declaring [extendable sources](./struct/extend.md).
+
+## Partial Attribute Forwarding
```

Comment:
I considered adding a section to the docs about how to use this functionality with `clap` for users who may want to integrate their configuration with CLI arguments. [It's a fairly popular request](https://github.com/clap-rs/clap/discussions/2763). Let me know if you'd be interested in that and I can add it here or in a separate PR.